js/modals/question-modal.js

```diff
@@ -189,15 +197,15 @@ $(function() {
    }
 
    function createQuestionForm() {
-      var $questionForm = $('<form></form>');
-      var questionName = '<fieldset class="form-group"> <label for="q-name-input">Question Name </label>' +
-          '<input type="text" id="q-name-input" class="form-control" placeholder="Question Name"></fieldset>';
-      var $questionType = $('<fieldset class="form-group"> <label for="q-type-select">Question Type </label></fieldset>');
-      var questionStatement = '<fieldset class="form-group"> <label for="q-statement-input">Question Statement</label>' +
-          '<textarea class="form-control" id="q-statement-input" rows="3" placeholder="Question Statement"></textarea></fieldset>';
-      var $typeSelect = $('<select class="form-control" id="q-type-select"><option>Multiple Choice</option>' +
-          '<option>Bug Fix</option><option>Missing Code</option><option>Complete Code</option></select>');
-      var $questionTypeArea = $('<div class="question-type-area multiple-choice"></div>');
+      var $templates = $('.qa-templates');
+
+      var $questionForm = $templates
```

Comment:
The variable names used here are very good and are a great example of what I mentioned in my other comment.